config.yaml

```diff
@@ -72,11 +72,16 @@ params:
         company: 2i2c
         details: |1-
           - **Designed, deployed, and maintained resilient cloud infrastructure** across AWS, Azure, and GCP, including production-grade Kubernetes clusters.
+          - **Designed and deployed a robust Python application** to automate backup creation and management for Google Filestore instances storing critical customer data, ensuring reliable disaster recovery in production environments.
+          - **Designed and deployed a Helm chart** to automate the creation and management of backups for Google Filestores containing customer data, ensuring reliable disaster recovery across production environments.
           - **Defined and implemented infrastructure as code and CI/CD pipelines** to automate deployments and improve system reliability and delivery speed.
+          - **Engineered Python-based automation** to detect file-level changes, enabling a smart CI/CD pipeline that selectively redeploys only impacted environments—significantly reducing deployment time and resource usage.
```

Comment:
highlight _significantly reducing deployment time and resource usage_

it makes engineers yummy.